tutorials/openmm-torch-nnpops.ipynb

```diff
@@ -0,0 +1,440 @@
+        "Covered topics:\n",
+        " * Installation of the software with [Conda](https://docs.conda.io/)\n",
+        " * Creation of an NNP (neural network potential) with [TorchANI](https://aiqm.github.io/torchani/)\n",
+        " * Acceleration of [ANI-2x](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/60c747e9567dfec574ec48df/original/extending-the-applicability-of-the-ani-deep-learning-molecular-potential-to-sulfur-and-halogens.pdf) with [NNPOps](https://github.com/openmm/NNPOps) \n",
+        " * Integration of [OpenMM](https://openmm.org/) and [PyTorch](https://pytorch.org/) and with [OpenMM-Torch](https://github.com/openmm/openmm-torch)\n",
+        " * Setup and execution of a simulation"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "BKG1ZNq94f-Z"
+      },
+      "source": [
+        "## Install Conda\n",
+        "\n",
+        "[Conda](https://docs.conda.io/) is a package and environment manager. On Google Colab, Conda is installed with [conda-colab](https://github.com/jaimergp/condacolab). On your computer, you should follow these [installation instructions](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).\n",
+        "\n",
+        "⚠️ Do not use conda-colab on your computer!"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "EcB2lc1s3ZXG",
+        "outputId": "1ce0bad8-a050-48d5-c18a-9e49fbeb11c2"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "⏬ Downloading https://repo.anaconda.com/miniconda/Miniconda3-py37_4.9.2-Linux-x86_64.sh...\n",
+            "📦 Installing...\n",
+            "📌 Adjusting configuration...\n",
+            "🩹 Patching environment...\n",
+            "⏲ Done in 0:00:21\n",
+            "🔁 Restarting kernel...\n"
+          ]
+        }
+      ],
+      "source": [
+        "!pip install -q condacolab\n",
+        "import condacolab\n",
+        "condacolab.install_miniconda()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tWlPu4KF5DFZ"
+      },
+      "source": [
+        "## Install software\n",
+        "\n",
+        "The [conda-forge](https://conda-forge.org/) channel is used for software.\n",
+        "\n",
+        "⚠️ The installation might take up to 10 min!"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 2,
+      "metadata": {
+        "id": "-qFE_cKF5958"
+      },
+      "outputs": [],
+      "source": [
+        "# Note: Remove \"mmh\" when NNPOps in available on conda-forge (https://github.com/openmm/NNPOps/issues/26)\n",
+        "# Note: \"cudatoollit=11.2\" is need to override a pinning in conda-colab\n",
+        "!conda install -q -c conda-forge -c mmh \\\n",
```

Comment:
If we use `mamba` instead of `conda` here, it can speed up installs significantly (presuming we also used mambaforge above).

@mikemhenry : Are we ready to drop the `-c mmh` yet?

Reply:
Not yet, if we can't drop it by the time we want to merge the tutorial, we can make an issue to drop it so I won't forget once we get NNPOps on conda-forge